app/controllers/three_d_secure_controller.js

```diff
@@ -117,6 +131,19 @@ module.exports = {
         postUrl: challengeUrl,
         worldpayChallengeJwt: worldpayChallengeJwt
       }
+
+      logger.info('Rendering form to post to card issuer', {
```

Comment:
In this case we know specifically that it’s a Worldpay 3DS Flex payment and therefore posting to Worldpay, so something like “Rendering form to post to Worldpay to initiate 3DS authentication” would be good.

Reply:
Done